MoreLinq/PartialSort.cs

```diff
@@ -44,6 +44,27 @@ public static IEnumerable<T> PartialSort<T>(this IEnumerable<T> source, int coun
         }
 
         /// <summary>
+        /// Combines <see cref="OrderBy{T, TKey}(IEnumerable{T}, Func{T, TKey}, IComparer{TKey}, OrderByDirection)"/>,
```

Comment:
The summary should mention a word about `direction` (without quoting the parameter). For example: “An additional parameter specifies the direction of the sort”. Ditto for other added overloads.